svgWriterFx.js

```diff
@@ -163,42 +163,59 @@
         };
         
         this.hasDropShadow = function (ctx) {
-            var omIn = ctx.currentOMNode,
-                dropShadow;
-            if (omIn && omIn.style && omIn.style.fx) {
-                dropShadow = omIn.style.fx.dropShadow;
-            
-                if (dropShadow && dropShadow.enabled) {
-                    return true;
-                }
+            var omIn = ctx.currentOMNode;
+            if (omIn && omIn.style && omIn.style.fx && omIn.style.fx.dropShadowMulti) {
```

Comment:
I actually planned to do some optimizations in this way to remove some functions.